app/src/main/java/com/mad41/tripreminder/historyListner.java

```diff
@@ -0,0 +1,11 @@
+package com.mad41.tripreminder;
+
+import com.mad41.tripreminder.room_database.trip.Trip;
+
+import java.util.List;
+
+public interface historyListner {
+    public void showNotes(List<Trip> notes , int id);
+    public void DeleteTrip(List<Trip> Trips , int id);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.